### ISSUE TITLE
Accessibility: Element is not keyboard-operable – <span> instead of buttons [2]

### ISSUE THREAD

**Issue by vikrampm1** [woothemes/woocommerce#51834]
**Prerequisites (mark completed items with an [x]):**
- [x] I have checked that my issue type is not listed here https://github.com/woocommerce/woocommerce/issues/new/choose
- [x] My issue is not a security issue, support request, bug report, enhancement or feature request (Please use the link above if it is).

### Issue Description:

The element(s) are not keyboard-focusable or clickable without a mouse because they are coded as <span> instead of HTML buttons.

![T007_KB_MissingKB](https://github.com/user-attachments/assets/779aebfc-4f15-449a-902a-3eb35d11c9f6)

### Component or Page: 

Account details My Account Page

### Name affected element:

Show/Password icon

### Relevant code snippets:

`<span class=""show-password-input""></span>`

### Recommended fix:

Instead of using the <span> element, it is recommended to use HTML <button> tags for these elements to make them inherently operable with keyboard.
 
 If you cannot replace a span with a button, it is possible to remediate it to function as a button with ARIA. The span would need to be given a button role and tabindex=""0"" to ensure that it can be focused with the keyboard. You would also need to write JavaScript to ensure that the span will function properly as a button for keyboard users. For more information on the aria button role, see: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/button_role
